controllers/articleLikesController.js

```diff
@@ -39,10 +35,15 @@ export default class ArticleLikesController {
           return res.status(CREATED).json({ message: "Successfully liked" });
         }
         if (articleLike.like) {
-          await articleLike.destroy();
-          return res.status(OK).json({ message: "Unliked successfully" });
+          await ArticleLike.destroy({
```

Comment:
@Musigwa After destroying article's like, how do you count the number of people that disliked the article?

Reply:
Hi @oesukam, although this file seems to contain a few changes from my PR, I'm not the one who created the articleLikes handlers. maybe you can give me some time to confirm with the team while you're reviewing the other files that contain the crucial work from this PR. Thanks for your patience, I'll get back to you.